App.tsx

```diff
@@ -1,24 +1,16 @@
+import NavigationStack from 'navigation/index';
```

Comment:
```suggestion
import React from 'react'
import { NavigationContainer } from '@react-navigation/native';
import { QueryClient, QueryClientProvider } from '@tanstack/react-query';

import NavigationStack from 'navigation/index';
```

We always import everything from React first, then 3rd party libraries or our own and then the other imports

Reply:
totally agree, I've forgot to add the rest of path alias